From ad52e4fe98ba7c19b42af737dadc1d0b1c848647 Mon Sep 17 00:00:00 2001
From: Hien Dang <hien.dang.eb@renesas.com>
Date: Fri, 8 Dec 2017 10:53:11 +0700
Subject: [PATCH 161/909] soc: renesas: rcar_ems_ctrl: Enable EMS for CA53

commit ab8ffb7b3efe52e7197a16164b15d07024d33d2c from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

In kernel v4.9, CPUFreq of CA53 has not been supported,
so EMS target cpus are only CA57 cores.
When upgrading to v4.14, CPUFreq of CA53 is supported.

This patch implements so that CA53 cores are also
EMS target cpus.

Signed-off-by: Hien Dang <hien.dang.eb@renesas.com>
Signed-off-by: Dien Pham <dien.pham.ry@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/renesas/rcar_ems_ctrl.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/soc/renesas/rcar_ems_ctrl.c b/drivers/soc/renesas/rcar_ems_ctrl.c
index 1b5be366ea06..8fda8f92c9db 100644
--- a/drivers/soc/renesas/rcar_ems_ctrl.c
+++ b/drivers/soc/renesas/rcar_ems_ctrl.c
@@ -257,11 +257,8 @@ static int rcar_ems_cpu_shutdown_init(void)
 
 	total_target_cpu = of_count_phandle_with_args(ems_node,
 						"target_cpus", 0);
-
 	for_each_online_cpu(cpu) {
 		tmp_node  = of_get_cpu_node(cpu, NULL);
-		if (!of_device_is_compatible(tmp_node, "arm,cortex-a57"))
-			continue;
 		for (i = 0; i < total_target_cpu; i++) {
 			cpu_node = of_parse_phandle(ems_node, "target_cpus", i);
 			if (tmp_node == cpu_node) {
-- 
2.17.1

